pkg/internal/ebpf/generictracer/generictracer.go

```diff
@@ -258,7 +258,7 @@ func (p *Tracer) KProbes() map[string]ebpfcommon.FunctionPrograms {
 			Required: true,
 			Start:    p.bpfObjects.KprobeTcpClose,
 		},
-		"tcp_sendmsg": {
+		"tcp_sendmsg_locked": {
```

Comment:
We needed this change since sock_msg attaches to tcp_sendmsg and our probes don't run at all, and we need them to check for various things. Attaching lower to the locked version of sendmsg works. We can't read the bvec buffers that sock_msg makes, but our logic still runs.